### PR TITLE
Added missing html redirects back in

### PR DESCRIPTION
**Pull request by bdemers**
Generated mostly from d91bb31599764210fddccb3f3de26eaaafe4783d and the following script (actual script not important but included here for record)

```bash
#!/bin/bash
REDIRECTS=$(find . -type f | grep '\.html'| grep -v /javadoc/ |grep -v java_api_sdk | xargs grep -l 'http-equiv="refresh" content="0; url=' |sort |uniq | xargs grep 'http-equiv="refresh" content="0' | sed 's_<meta http-equiv="refresh" content="0; url=__' | sed 's_">$__' | sed 's_https://developer.okta.com__')

DEST_DIR="$HOME/dev/source/okta/blog/_source/_docs/"

while read -r line; do
    IFS=':  ' read -r -a ITEM <<< "$line"
    CURRENT_FILE="${DEST_DIR}/$(echo ${ITEM[0]} | sed 's_\.html_\.md_')"
    NEW_FILE="${ITEM[1]}"
    echo "$CURRENT_FILE"
    
    mkdir -p $(dirname $CURRENT_FILE)
    
cat > "$CURRENT_FILE" <<EOF
---
redirect_to: ${NEW_FILE}
---
EOF

done <<< "$REDIRECTS"
```

### CODE REVIEW

File: _source/_docs/guides/scim_developer_program.md [hunk 1]
```diff
@@ -1,3 +1,3 @@
 ---
-redirect_to: /standards/SCIM/scim_developer_program
+redirect_to: /standards/SCIM/index
```
Comment (jmelberg-okta) [inline _source/_docs/guides/scim_developer_program.md]: Can this just be: `standards/SCIM/`?

File: _source/_docs/api/rest/search-beta.md [hunk 2]
```diff
@@ -1,3 +1,3 @@
 ---
-redirect_to: /docs/api/resources/search-beta
+redirect_to: /docs/api/resources
```
Comment (jmelberg-okta) [inline _source/_docs/api/rest/search-beta.md]: I think this is safe to remove. Anything with `/rest` was been updated > 1 year ago to new routes.

File: _source/_docs/guides/scim_developer_program.md (hunk 1 above)
Comment (bdemers) [reply _source/_docs/guides/scim_developer_program.md]: Yes, but we already had another existing redirect that used `/index`, so i wanted to keep them the same.

File: _source/_docs/api/rest/search-beta.md (hunk 2 above)
Comment (bdemers) [reply _source/_docs/api/rest/search-beta.md]: 👍

File: _source/_docs/api/rest/apps.md [hunk 3]
```diff
@@ -0,0 +1,3 @@
+---
+redirect_to: /docs/api/resources/apps.html
```
Comment (jmelberg-okta) [inline _source/_docs/api/rest/apps.md]: All the `/rest` paths can be removed. @mystiberry-okta and I talked about this a few months back.

**Pull request merged**